docs/installation.md

```diff
@@ -8,7 +8,7 @@ The following instructions are for setting up a version of Amundsen using Docker
    ```bash
    $ git clone --recursive git@github.com:amundsen-io/amundsen.git
    ```
-3. Enter the cloned directory and run:
+3. Enter the cloned directory and run below. If it's your first time, you may want to proactively go through [troubleshooting](troubleshooting) steps, especially the first one related to heap memory for ElasticSearch and Docker engine memory allocation (leading to Docker error 137):
```

Comment:
Suggest putting this immediately below the code block

Reply:
Thanks updated!